nomination/templates/nomination/url_add.html

```diff
@@ -8,6 +8,17 @@
 
 {% block title %}Nomination Tool: Add/Edit a Site{% endblock %}
 
+{% block localized_styles %}
+    {% if metadata_vals %}
+        {% for met,vals in metadata_vals %}
+            {% if met.form_type == "select" or met.form_type == "selectsingle" %}
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/slim-select/1.26.0/slimselect.min.js"></script>
+                <link href="https://cdnjs.cloudflare.com/ajax/libs/slim-select/1.26.0/slimselect.min.css" rel="stylesheet"></link>
+            {% endif %}
+        {% endfor %}
+    {% endif %}
+{% endblock %}
+
```

Comment:
It seems if there are multiple select fields, these links will be included multiple times here and in url_listing.html.

Reply:
ah, right. :man_facepalming: